comp/core/secrets/secretsimpl/secrets.go

```diff
@@ -47,6 +48,14 @@ func Module() fxutil.Module {
 		fx.Provide(newSecretResolverProvider))
 }
 
+type secretContext struct {
+	// origin is the configuration name where a handle was found
+	origin string
+	// path is the key associated to the secret in the YAML configuration.
+	// Example: in this yaml: '{"token": "ENC[token 1]"}', 'token' is the path and 'token 1' is the handle.
+	path []string
```

Comment:
We changed from a `string` to a `[]string.` Does the comment above need to be updated?

Reply:
Good suggestion, updated the example to demonstrate the distinction.